statics/js/components/graph-layout.js

```diff
@@ -1210,7 +1216,8 @@ TopologyGraphLayout.prototype = {
     var linkEnter = this.link.enter()
       .append("path")
```

Comment:
In order to ease the click

I think it could be nice to add a G container here using CSS "pointer-events: bounding-box" on it to capture onmouseover. In that case we could increase the stroke-width.